### PR TITLE
Added SemCor version2.1

### PR DESCRIPTION
**Pull request by tejasvaidhyadev**
As per #23 SemCor2.1 is added

### CODE REVIEW

**Comment by oxinabox** [comment]
@Ayushk4  can you review?
I have lost track of the context of this one

**Comment by Ayushk4** [comment]
Sure, I will do it tomorrow morning.

File: src/SemCor_DataDeps.jl [hunk 1]
```diff
@@ -5,6 +5,7 @@ for (ver, checksum) in [("1.6", "16814254fe194d55a2fcc24858aa76d71de3c49e495bd98
             ("1.7", "0495577ac3a87c2a64fe6189798ea046de0f44943dfb7b60fe38cf648d34c421"),
             ("1.7.1", "70b9eb7ca0dc9d67655f9d671d40be10aeff490f0bea4f10cb1946127b74c102"),
             ("2.0", "93fbae725f0125dedb7369403fda1dace85b2dcd8a523ed80af23e863b18ef2c"),
+	    ("2.1", "0714f07dbcb84a215d668f3ee85892fa8fa4a8154439662eb7529413367b8f56"),
```
Comment (Ayushk4) [inline src/SemCor_DataDeps.jl]: The indentation seems a bit off. Perhaps because of the usage of both spaces and tabs.

Otherwise, LGTM.

Comment (Ayushk4) [reply src/SemCor_DataDeps.jl]: Resolved in 04bb500

**Comment by Ayushk4** [comment]
@oxinabox I think this is good to go. Should I merge it?

**Comment by oxinabox** [comment]
Sure, and tag a release

File: src/SemCor.jl [hunk 2]
```diff
@@ -1,7 +1,48 @@
 struct SemCor{S}
     filepaths::Vector{S}
 end
+"""
```
Comment (oxinabox) [inline src/SemCor.jl]:
```suggestion

"""
```

**Pull request merged**